src/pe/header.rs

```diff
@@ -163,14 +262,24 @@ impl CoffHeader {
     }
 
     /// Return the COFF symbol table.
-    pub fn symbols<'a>(&self, bytes: &'a [u8]) -> error::Result<symbol::SymbolTable<'a>> {
+    pub fn symbols<'a>(&self, bytes: &'a [u8]) -> error::Result<Option<symbol::SymbolTable<'a>>> {
```

Comment:
NB: another breaking change; I assume were just returned an empty symbol table before if the offset was 0? Or maybe this never came up in the various pe binaries i tested on?

Reply:
I believe it never came up in the PE binaries you tested on but this function usually crashed in my experience when I called it.

Reply:
Maybe it doesn't crash anymore if we had the Scroll patch for the zero-sized types :P.

Reply:
Hmm are you saying we should construct an empty symbol table and not an optional in this case? If yes, it might be worthwhile to not make this optional, as it might be changed back if we had the option of returning an empty symbol table / have scroll not crash on offset == 0

Reply:
I guess what i'm saying does the optionality signify important _semantic_ information to the user that the thing you are trying to get might not "be there", as opposed to working around a scroll bug. If 0 signifies it isn't "there"/it isn't required, then let's keep it optional, yes.

Reply:
It's definitely the case that the optionality convey important semantic information IMHO, I suggest to keep it.